### PR TITLE
V1->V2 connect calendar

### PR DESCRIPTION
**Pull request by sean-brydon**
## What does this PR do?

https://github.com/calcom/cal.com/pull/3891

Puts these changes from V1 into V2 components

### CODE REVIEW

File: apps/web/pages/v2/event-types/index.tsx [hunk 1]
```diff
@@ -609,6 +614,22 @@ const EventTypesPage = () => {
...
+                      You can connect your calendar from
+                      <a href="/apps/categories/calendar" className="underline">
+                        here
+                      </a>
+                      .
+                    </Trans>
+                  }
+                />
+              )}
               {data.eventTypeGroups.map((group, index) => (
```
Comment (PeerRich) [inline apps/web/pages/v2/event-types/index.tsx:633]: this didnt seem to work in the old PR and was reverted, are we sure we wanna keep this?

Comment (sean-brydon) [reply apps/web/pages/v2/event-types/index.tsx:633]: @alannnc  - Is it worth removing this?

Comment (alannnc) [reply apps/web/pages/v2/event-types/index.tsx:633]: Just take whats on main and you should be ready to go. I think on main its called as <NoCalendarConnectedAlert/>

File: apps/web/pages/v2/event-types/index.tsx [hunk 2]
```diff
@@ -634,4 +655,26 @@ const EventTypesPage = () => {
...
+    });
+    defaultCalendarConnected = !!defaultCalendar;
+  }
+  return {
+    props: {
+      defaultCalendarConnected,
+    },
+  };
+}
+
```
Comment (alannnc) [inline apps/web/pages/v2/event-types/index.tsx:679]: With new component this is not required now.

**Review by alannnc — APPROVED** [review]

**Pull request merged**